src/components/learner-credit-management/BudgetDetailCatalogTabContents.jsx

```diff
@@ -15,7 +15,7 @@ import { configuration } from '../../config';
 const BudgetDetailCatalogTabContents = () => {
   const history = useHistory();
   const { location } = history;
```

Comment:
do we still need `location` now?

Reply:
removed in favor of using the `useLocation` location object 👍🏽